jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -78,7 +78,13 @@
       </v-row>
 
       <v-row v-for="(item, index2) in region" class="row-no-outside-padding">
-        <v-text-field
+        <v-text-field v-if="item.name === 'Parent'"
+          :label="item.name"
+          :value="item.value"
+          style="padding-top: 0px; margin-top: 0px"
+          :readonly="true"
```

Comment:
Per discussion in tag up/code review:
```suggestion
          :readonly="true"
          hint="Subset was defined with respect to this reference data (read-only)"
```